Source/WebCore/loader/FrameLoader.h

```diff
@@ -536,6 +538,8 @@ class FrameLoader final : public CanMakeCheckedPtr<FrameLoader> {
     bool m_shouldRestoreScrollPositionAndViewState { false };
 
     bool m_errorOccurredInLoading { false };
+
+    bool m_isInitialAboutBlank { true };
```

Comment:
This seems like a flag that should be on a document loader. Why does the frame loader need to copy the flag out?

Reply:
You are right, a flag on DocumentLoader or Document is better, done.